docs/ADCore/NDFileHDF5.rst

```diff
@@ -751,45 +751,9 @@ Parameters and Records
     - longout, longin
   * - asynInt32
     - r/w
-    - Size of extra dimension 3
-    - HDF5_extraDimSize3
-    - $(P)$(R)ExtraDimSize3, $(P)$(R)ExtraDimSize3_RBV
-    - longout, longin
-  * - asynInt32
-    - r/w
-    - Size of extra dimension 4
-    - HDF5_extraDimSize4
-    - $(P)$(R)ExtraDimSize4, $(P)$(R)ExtraDimSize4_RBV
-    - longout, longin
-  * - asynInt32
-    - r/w
-    - Size of extra dimension 5
-    - HDF5_extraDimSize5
-    - $(P)$(R)ExtraDimSize5, $(P)$(R)ExtraDimSize5_RBV
-    - longout, longin
-  * - asynInt32
-    - r/w
-    - Size of extra dimension 6
-    - HDF5_extraDimSize6
-    - $(P)$(R)ExtraDimSize6, $(P)$(R)ExtraDimSize6_RBV
-    - longout, longin
-  * - asynInt32
-    - r/w
-    - Size of extra dimension 7
-    - HDF5_extraDimSize7
-    - $(P)$(R)ExtraDimSize7, $(P)$(R)ExtraDimSize7_RBV
-    - longout, longin
-  * - asynInt32
-    - r/w
-    - Size of extra dimension 8
-    - HDF5_extraDimSize8
-    - $(P)$(R)ExtraDimSize8, $(P)$(R)ExtraDimSize8_RBV
-    - longout, longin
-  * - asynInt32
-    - r/w
-    - Size of extra dimension 9
-    - HDF5_extraDimSize9
-    - $(P)$(R)ExtraDimSize9, $(P)$(R)ExtraDimSize9_RBV
+    - Size of the N'th extra dimension, for N in [3..9]
+    - HDF5_extraDimSize[N]
+    - $(P)$(R)ExtraDimSize[N], $(P)$(R)ExtraDimSize[N]_RBV
```

Comment:
Maybe there should be an example? It's not clear from this how the `[N]` substitution works. Is it `[3]` or `3`? I'd like to use `$N`, but I think that's confusing because `$(P)$(R)` is already in use here; maybe `<N>` would be better, to avoid confusion with the array notation from above?